saleor/graphql/core/mutations.py

```diff
@@ -269,8 +270,10 @@ def check_permissions(cls, context, permissions=None):
         if context.user.has_perms(permissions):
             return True
         service_account = getattr(context, "service_account", None)
-        if service_account and service_account.has_perms(permissions):
-            return True
+        if service_account:
+            if AccountPermissions.MANAGE_STAFF in permissions:
+                return False
+            return service_account.has_perms(permissions)
```

Comment:
As a user, I give `Manage Staff` permission to my service account, then I try to manage staff, it doesn't work for some reasons. I open a ticket about this bug.

I believe it is a bad way of doing.

On top, a user, I would like to be able to manage the staff to perform automatic actions, such as adding new employees. Maybe putting a site settings or something that only shop owners can set?

Reply:
Hm, during some daily or PR meetings we decided to don't allow service account to handle operations/queries which are protected by `MANAGE_STAFF`, so I am not sure it should be handle in settings. But maybe further discussion is needed. @maarcingebala @patrys

Reply:
@NyanKiyoshi It will be added in the future, for now we want to disable managing staff for service accounts.

Reply:
Shouldn't we remove `Manage Staff` from the services?